src/iio_wrapper.cpp

```diff
@@ -120,19 +177,36 @@ IIOWrapper::IIOWrapper()
 IIOWrapper::~IIOWrapper()
 {
     iio_context_destroy(m_object_context);
```

Comment:
It doesn't seem that m_object_context is being used anymore. Destroying it is unnecessary.